src/Web/Sfa.Das.Sas.Infrastructure/Repositories/AssessmentOrganisationApiRepository.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using SFA.DAS.Apprenticeships.Api.Types.AssessmentOrgs;
+using SFA.DAS.AssessmentOrgs.Api.Client;
+
+namespace Sfa.Das.Sas.Infrastructure.Repositories
+{
+    public sealed class AssessmentOrganisationApiRepository : IAssessmentOrgsApiClient
+    {
+        private readonly IAssessmentOrgsApiClient _apiClient;
+
+        public AssessmentOrganisationApiRepository(IAssessmentOrgsApiClient apiClient)
+        {
+            _apiClient = apiClient;
+        }
+
+        public void Dispose()
+        {
+            _apiClient.Dispose();
+        }
+
+        public Organisation Get(string organisationId)
+        {
+            var res = _apiClient.Get(organisationId);
+            return res;
+        }
```

Comment:
can be 1 liners like 
`public Organisation Get(string organisationId)
{
   return _apiClient.Get(organisationId);
}`